test/http-test.js

```diff
@@ -217,7 +217,7 @@ describe('HTTP', function() {
         + '0000000000000000000000000000000000000000000000000000000000000000'
         + 'ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff',
       curtime: json.curtime,
-      mintime: 1548148526,
+      mintime: 1554268736,
```

Comment:
This is the value in the genesis block: https://github.com/handshake-org/hsd/blob/af86d4899791601d10212115c80a2f2136fdb004/lib/protocol/genesis.js#L109

No blocks were mined so we can expect the above + 1 because of this line: https://github.com/handshake-org/hsd/blob/af86d4899791601d10212115c80a2f2136fdb004/lib/node/rpc.js#L1492